lib/watir.rb

```diff
@@ -14,58 +14,55 @@
 require 'watir/after_hooks'
 
 module Watir
+  @relaxed_locate = true
   @always_locate = true
 
+  attr_writer :relaxed_locate, :always_locate, :default_timeout, :prefer_css, :locator_namespace
+
   class << self
-    def always_locate?
-      @always_locate
+    #
+    # Whether or not Watir should wait for an element to be found or present
+    # before taking an action.
+    # Defaults to true.
+    #
+
+    def relaxed_locate?
+      @relaxed_locate
     end
 
     #
     # Whether or not Watir should cache element references or always re-locate an Element on use.
     # Defaults to true.
     #
 
-    def always_locate=(bool)
-      @always_locate = bool
-    end
-
-    def default_timeout
-      @default_timeout ||= 30
+    def always_locate?
+      @always_locate
     end
 
     #
     # Default wait time for wait methods.
     #
 
-    def default_timeout=(value)
-      @default_timeout = value
-    end
-
-    def prefer_css?
-      @prefer_css
+    def default_timeout
+      @default_timeout ||= 30
```

Comment:
These are implicit waits in a way that if element is not present, it waits for it to become present automatically.